src/fragments/lib/push-notifications/android/record_notifications/20_record_notification_received.mdx

```diff
@@ -0,0 +1,114 @@
+You can record receipt of a push notification with Amazon Pinpoint by calling `Amplify.Notifications.Push.recordNotificationReceived()` from the Messaging Service class.
+
+<Callout>
+
+You should call either `handleNotificationReceived` or `recordNotificationReceived`, but not both since `handleNotificationReceived` already calls `recordNotificationReceived`.
```

Comment:
```suggestion
You should call either `handleNotificationReceived` or `recordNotificationReceived`, but not both since `handleNotificationReceived` internally calls `recordNotificationReceived`.
```